yarn.lock

```diff
@@ -11062,10 +11062,10 @@ jest-cli@^27.5.1:
     prompts "^2.0.1"
     yargs "^16.2.0"
 
-jest-config@^27.5.1:
+jest-config@^27.5.1, "jest-config@npm:@amoo-miki/jest-config@27.5.1":
```

Comment:
do we want to used `osd-pm` to build out packages so that we can source control in this repo and not need to publish from a personal account?

Reply:
While we could, I was hoping to use that option for long-term packages we want to own. This one will go away as soon as we get rid of Angular and we can use `jest-circus`.

Reply:
Is it clear how long it will be before Angular is removed? I don't see a clear timeline or outline of how much work is left in https://github.com/opensearch-project/OpenSearch-Dashboards/issues/1560.

Reply:
Yeah, unfortunately the amount of work left in the deangularization is not trivial. I could be swayed either way, but I lean toward Rocky's suggestion of adding it to `packages` instead.

Reply:
do we want to try that?

Reply:
Donno how trivial that will be. Will try later.